RandomWordGen.java

```diff
@@ -0,0 +1,22 @@
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Map;
+import java.util.HashMap;
+import java.util.stream.Stream;
+import java.lang.Math;
+
+public class RandomWordGen {
+    public static void main(String[] args) throws IOException {
+        int random = (int)(Math.random()*10);
+        String delimiter = Integer.toString(random);
+        Map<String, String> map = new HashMap<>();
+
+        try(Stream<String> lines = Files.lines(Paths.get("/Users/Zeus/Documents/JavaBookClub/Dictionary.txt"))){
```

Comment:
To get the lines from a file, you can use Files.readAllLines (from @christopher-brady)